app/src/main/java/com/macbitsgoa/events/maps/Venue.java

```diff
@@ -0,0 +1,33 @@
+package com.macbitsgoa.events.maps;
+
+class Venue {
+
+    private final String latitude;
```

Comment:
make all model attributes `public`
make lat and long `float`
remove getters and setters